### DISCUSSION TITLE
Client Showing of Mapping Notifications

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#27202] [Ideas]
Would be nice to be able to see the mapper notifications in-game... it's a bit cumbersome to view them in the browser and having to find it in-game to download.

**Comment by Rekunan** [comment]
Should also apply to the other possible watchlist notifs 
![image](https://github.com/ppy/osu/assets/24501632/69d75c43-b3cc-4452-b340-3b47c16d7af8)

**Comment by bdach** [comment]
So you're asking for that screen to be reimplemented in client, correct?

Not sure we have the API endpoints for this but it's probably feasible.

**Reply by 19peytonsmith** [reply]
That would be nice yeah... even if there's a temporary solution of just redirecting to browser to view, that's better than nothing. It doesn't help though that when I download beatmaps from browser it opens stable client and not lazer though 😅 that's probably my main motivation to wanting the ability to download beatmaps from my notifications in-game.

**Reply by bdach** [reply]
> when I download beatmaps from browser it opens stable client and not lazer though

should be sorted next release (https://github.com/ppy/osu/pull/27001)
